### DISCUSSION TITLE
Bug? Unable to delete fields / API namespaces

### DISCUSSION THREAD

**Discussion by JoeMatkin** [swellstores/community#476] [Help] [answered]
I have tried to delete the "packaging" fields shown in the below screenshot, and they show as deleted under the "Content field" tab under the product model. But when navigating to data fields, they can still be seen.
<img width="1222" alt="Screenshot 2024-01-16 at 10 47 44" src="https://github.com/swellstores/community/assets/72705313/22129b57-9922-42dc-9e76-1e4f5fb0962a">

This is the "packaging" field I have tried to delete and as you can see they are still here despite being deleted under "content fields".
<img width="1252" alt="Screenshot 2024-01-16 at 10 46 53" src="https://github.com/swellstores/community/assets/72705313/06727c6c-db76-4b45-a72f-aca4ff887da7">

**Comment by steven-k1** [comment]
@JoeMatkin Looking into this

**Reply by logeshswell** [reply]
@JoeMatkin Have you tried refreshing the dashboard after deleting the field under the **Content fields** tab?
It should have gone. Can you check and confirm?

**Comment by steven-k1** [comment, marked as answer]
@JoeMatkin A fix has been deployed, and the corresponding data fields should now be deleted when the custom content field is deleted.
